gadgets/snapshot_process/artifacthub-pkg.yml

```diff
@@ -0,0 +1,32 @@
+# Artifact Hub package metadata file
+# This file has the automatically generated from gadget.yaml
+version: 0.1.0
```

Comment:
I am thinking how should versioning be handled? Like should this be the latest stable release or something else? I will translate this version as `ig run snapshot_process:v0.1.0`  which is broken i guess.

Reply:
I don't know how to handle it. For now, this is not managed automatically. So users have to update their `artifacthub-pkg.yml` file manually, which is impractical. But I suggest to handle this in a future version.

Note that it is not related to the tag of the OCI image. This still use `latest` because that's what I wrote in the `containersImages` section:
```
containersImages:
    - name: gadget
      image: "ghcr.io/inspektor-gadget/gadget/snapshot_process:latest"
      platforms:
        - linux/amd64
        - linux/arm64
```

Note that it is not possible to specify several entries in `containersImages` for each version of a gadget. If a user wants to declare several versions to Artifact Hub, they would have to write several `artifacthub-pkg.yml` files. See [this check](https://github.com/artifacthub/hub/blob/7aee37bd1ffde48637d4db23a74a8e115dc16d80/internal/pkg/metadata.go#L260-L265).

Reply:
I added a commit to automatically update the version after a release.